api/v1alpha1/channel_types.go

```diff
@@ -24,12 +24,16 @@ import (
 // ChannelSpec defines the desired state of Channel
 type ChannelSpec struct {
 	// Name of the slack channel
+	// +required
 	Name string `json:"name"`
 
 	// Make the channel private or public
+	// +required
 	Private bool `json:"private,omitempty"`
```

Comment:
This will be optional because if a user doesn't specify it's value, it will be false(public) by default